edk2toolext/invocables/edk2_platform_build.py

```diff
@@ -60,10 +64,17 @@ class Edk2PlatformBuild(Edk2Invocable):
 
     def AddCommandLineOptions(self, parserObj):
         ''' adds command line options to the argparser '''
+    # check to see if the settings manager has uefibuilder
+        uefi_build_class = self.PlatformSettings.GetUefiBuilderClass()
 
         # PlatformSettings could also be a subclass of UefiBuilder, who knows!
         if isinstance(self.PlatformSettings, UefiBuilder):
             self.PlatformBuilder = self.PlatformSettings
+        elif uefi_build_class:
```

Comment:
This expects that a non-implementation would return None, right? Just confirming I understand the flow.

Reply:
Yes. Non-implementations would return None